.github/workflows/push-package-solarwinds.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2023 SolarWinds, LLC.
+# All rights reserved.
+
+name: Ruby Gem to Github Package
+
+on:
+  workflow_dispatch:
+    inputs:
+      rubygem-path:
+        required: true
+        description: 'The path that package you want to publish (should include /)'
+        # metrics_sdk/
+        # metrics_api/
+        # exporter/otlp/
+
+      rubygem-name:
+        required: true
+        description: 'The name that package you want to publish'
+        # opentelemetry-metrics-sdk
+        # opentelemetry-metrics-api
+        # opentelemetry-exporter-otlp
```

Comment:
Can you put in an actual gem name from our fork that we'd publish in the suggestion above, and remove these comments which don't apply to this repo?